gridstatus/tests/test_ercot.py

```diff
@@ -32,6 +32,14 @@ def test_get_as_prices(self):
         assert df.columns.tolist() == as_cols
         assert df["Time"].unique()[0].date() == date
 
+        date = pd.Timestamp(2022, 11, 8).date()
+        today = utils._handle_date("today").date()
+        df = self.iso.get_as_prices(date, end="today")
+        assert df.shape[0] >= 0
+        assert df.columns.tolist() == as_cols
+        assert df.Time.min().date() == date
+        assert df.Time.max().date() == today
```

Comment:
i think it's worth also checking to make sure every day between `date` and `today` is also defined in the response